### PR TITLE
Safer mixpanel check, send identity to Oilbird, Pelican can alias.

### PR DESCRIPTION
**Pull request by rkuykendall**

### CODE REVIEW

File: src/analytics/actions.es6 [hunk 1]
```diff
@@ -58,6 +58,27 @@ function tryEmail() {
...
+    return id;
+  }
+
+  // If logged out, set the id to the mixpanel ID,
+  if (is.function(_.get(window, 'mixpanel.get_distinct_id'))) {
+    return window.mixpanel.get_distinct_id();
+  }
+
+  // If we can't find mixpanel, fallback to null ID
+  return id;
```
Comment (diezguerra) [inline src/analytics/actions.es6]: how is this null? this could be anything, potentially -- whatever is passed

File: src/analytics/actions.es6 [hunk 2]
```diff
@@ -194,8 +200,8 @@ function alias(to, from, options, fn) {
     }
 
     // See Mixpanel rules in identify function
-    if (appInfo.app != 'tailorbird') {
-        log('Alias should only be called on account creation.');
+    if (appInfo.app != 'tailorbird' && appInfo.app != 'pelican') {
```
Comment (diezguerra) [inline src/analytics/actions.es6]: make sure pelican has appInfo.app and it equals to that, if you haven't. This convention isn't spread throughout the repos I suspect

Comment (diezguerra) [inline src/analytics/actions.es6]: make sure pelican has appInfo.app and it equals to that, if you haven't. This convention isn't spread throughout the repos I suspect

**Comment by diezguerra** [comment]
:+1: minus comments

File: src/analytics/actions.es6 (hunk 2 above)
Comment (rkuykendall) [inline src/analytics/actions.es6]: Yeah, I'm planning on checking that when I add these aliases. Right now they don't exist. Actually a lot of our apps are missing that info. It's very easy to add though.

File: src/analytics/actions.es6 (hunk 1 above)
Comment (rkuykendall) [inline src/analytics/actions.es6]: :+1:

**Pull request merged**